src/main/resources/reference.conf

```diff
@@ -8,6 +8,58 @@ jicofo {
     // The assumed average packet rate per participant.
     average-participant-packet-rate-pps=500
   }
+  // Configure the codecs and RTP extensions to be used in the offer sent to clients.
+  codec {
+    vp8 {
+      enabled=true
+      pt=100
+      // Payload type for the associated RTX stream. Set to -1 to disable RTX.
+      rtx-pt=96
+    }
+    vp9 {
+      enabled=true
+      pt=101
+      // Payload type for the associated RTX stream. Set to -1 to disable RTX.
+      rtx-pt=97
+    }
+    h264 {
+      enabled=true
+      pt=107
+      // Payload type for the associated RTX stream. Set to -1 to disable RTX.
+      rtx-pt=99
+    }
+
+    // RTP header extensions
+    rtp-extensions {
+      tof {
```

Comment:
Do you think it's even worth adding to new config?

Reply:
I just added them all, I don't think it hurts